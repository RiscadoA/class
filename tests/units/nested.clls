proc main() {
    cut {
        cut {
            println("wait x");
            wait x;
            println("wait y");
            wait y;
            println("end");
            ()
            |y:close|
            println("close y");
            close y
        }
        |x:close|
        println("close x");
        close x
    }
};;
