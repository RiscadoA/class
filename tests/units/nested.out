close x
close y
wait x
wait y
end
